tutorials/napps/restful_statistics.rst

```diff
@@ -196,7 +198,7 @@ humans to understand the data. Let's take a closer look at some lines:
 *Bps* and *pps* mean bytes per second and packets per second, respectively.
 As we are sending one ping every 0.2 seconds, *pps* will be close to 5.
 
-In the Mininet console, hit ``ctrl+c`` and run ``h1 ping -i 0.5 h2``.
+In the Mininet console, hit ``ctrl+c`` and run ``h1 ping -i 0,5 h2``.
```

Comment:
Since you have already added the note, and this is a very specific case for non en_US. IMO, it is better to keep as it was.